packages/appium-tizen-tv-driver/lib/driver.js

```diff
@@ -509,16 +508,12 @@ class TizenTVDriver extends BaseDriver {
       );
     }
 
-    if (this.opts.sendKeysStrategy === RC_TEXT_STRAT) {
+    if (this.#isRemoteRcMode && this.opts.sendKeysStrategy === RC_TEXT_STRAT) {
       if (Array.isArray(text)) {
         text = text.join('');
       }
-      await B.delay(800);
```

Comment:
i think i had this here when i was trying to make things stable, for some reason it wasn't just working on its own without some resting